calibrate.py

```diff
@@ -92,6 +93,102 @@
+    },
+    'squareSizeCm': {
+        'flag': "-s", 'flag_long': "--squareSizeCm", 'default': 2.5, 'type': float, 'min': 2.2,
+        'label': "Square size (cm)", 'help': "Square size of calibration pattern used in centimeters. Default: 2.0cm."
+    },
+    'markerSizeCm': {
+        'flag': "-ms", 'flag_long': "--markerSizeCm", 'default': None, 'type': float, 
+        'label': "Marker size (cm)", 'help': "Marker size in charuco boards."
+    },
+    'defaultBoard': {
+        'flag': "-db", 'flag_long': "--defaultBoard", 'default': False, 'type': bool, 'action': "store_true", 
+        'help': "Calculates the -ms parameter automatically based on aspect ratio of charuco board in the repository"
+    },
+    'squaresX': {
+        'flag': "-nx", 'flag_long': "--squaresX", 'default': 11, 'type': int, 'min': 1,
+        'label': "No. squres X", 'help': "number of chessboard squares in X direction in charuco boards."
+    },
+    'squaresY': {
+        'flag': "-ny", 'flag_long': "--squaresY", 'default': 8, 'type': int, 'min': 1,
+        'label': "No. squares Y", 'help': "number of chessboard squares in Y direction in charuco boards."
+    },
+    'rectifiedDisp': {
+        'flag': "-rd", 'flag_long': "--rectifiedDisp", 'default': True, 'type': bool, 'action': "store_false", 
+        'label': "Show rectified lines", 'help': "Display rectified images with lines drawn for epipolar check"
+    },
+    'disableRgb': {
+        'flag': "-drgb", 'flag_long': "--disableRgb", 'default': False, 'type': bool, 'action': "store_true", 
+        'label': "Disable RGB calibration", 'help': "Disable rgb camera Calibration"
+    },
+    'swapLR': {
+        'flag': "-slr", 'flag_long': "--swapLR", 'default': False, 'type': bool, 'action': "store_true",
+        'label': "Swap left and right cameras", 'help': "Interchange Left and right camera port."
+    },
+    'mode': {
+        'flag': "-m", 'flag_long': "--mode", 'default': ['capture', 'process'], 'nargs': '*', 'type': str, 
+        'help': "Space-separated list of calibration options to run. By default, executes the full 'capture process' pipeline. To execute a single step, enter just that step (ex: 'process')."
+    },
+    'invert_v': {
+        'flag': "-iv", 'flag_long': "--invertVertical", 'default': False, 'type': bool, 'action': "store_true", 
+        'label': "Flip verticaly", 'help': "Invert vertical axis of the camera for the display"
+    },
+    'invert_h': {
+        'flag': "-ih", 'flag_long': "--invertHorizontal", 'default': False, 'type': bool, 'action': "store_true", 
+        'label': "Flip horizontaly", 'help': "Invert horizontal axis of the camera for the display"
+    },
+    'outputScaleFactor': {
+        'flag': "-osf", 'flag_long': "--outputScaleFactor", 'default': 0.5, 'type': float, 'min': 0.1, 'max': 2.0,
+        'label': "Visualization scale factor", 'help': "Set the scaling factor for output visualization. Default: 0.5."
```

Comment:
Scaling arrows doesn't work on this one. 
<img width="446" alt="image" src="https://user-images.githubusercontent.com/43363595/211790600-44f008f8-7d34-4c25-82d1-144e60d859f2.png">